modules/user_data/templates/03_gdb_conf_overrides.sh.tpl

```diff
@@ -60,4 +60,21 @@ cat << EOF > /etc/systemd/system/graphdb.service.d/overrides.conf
 Environment="GDB_HEAP_SIZE=$${JVM_MAX_MEMORY}g"
 EOF
 
+parameters=$(aws ssm describe-parameters --query "Parameters[].Name" --output text)
```

Comment:
I see that you use region and connect timeout in the next commands, maybe use them here as well?